client/src/components/formHelpers/form-fields.tsx

```diff
@@ -59,13 +59,23 @@ function FormFields(props: FormFieldsProps): JSX.Element {
         validationError = (err as { message?: string })?.message;
       }
     }
-    const validationWarning = composeValidators(
-      name === 'githubLink' || isEditorLinkAllowed ? null : editorValidator,
-      fCCValidator,
-      httpValidator,
-      isLocalLinkAllowed ? null : localhostValidator,
-      pathValidator
-    )(value);
+
+    // If editor link is not allowed, use editor validator
+    // If local link is not allowed, use local validator
+    // If local link is allowed, use path validator
+    // Always used fCCValidator and httpValidator
```

Comment:
```suggestion
```
Sorry to be a pain about this, but this isn't the same as the logic below. Also, it's describing how the code works, which is what the code does.

It would be helpful to explain _why_ particular validators are used, next to the code that adds them, but it's better to say nothing than to include a misleading comment.